### ISSUE TITLE
Craftable external percursion cap, black gunpowder revolvers?

### ISSUE THREAD

**Issue by RodriguesCIA** [cataclysmbnteam/Cataclysm-BN#1516]
There's already a handmade revolver, but it fires .40 with metal casings, which makes it dependable on having some of that ammo.

How about a handmade version of the antique revolvers, using their paper cartridges? Their craft-able ammo would make them an asset for arming followers or just stocking up confidently ammo, better than playing around with the modern gun's low gunpowder tolerance, and would not completely fase out the triviliaty of flintlocks, as making percursion caps for the ammo would demand some tech, namely, a chisel and oxidizer powder.

They would, of course, be generally inferior to modern guns (revolvers or gas operated pistols), and wouldn't pack the same punch as flintlocks, and could be ballistic inferior to anitque/replica revolvers, but would be able to fire multiple shots before reload with the damage comparable to a big pistol round, would take quite some using before the fouling would start causing damage, and would be easier to replace before you have the capabilities to repair guns (firearsm/gunsmith repair kit).

**Comment by chaosvolt** [comment]
Adding makeshift percussion cap revolvers would be a neat change, yeah. Only issue is that blackpowder can be hard to obtain, if not for that then the bullet pulling updates from https://github.com/cataclysmbnteam/Cataclysm-BN/pull/886 would make percussion cap rounds WAY simpler.

Porting over makeshift overpressure rounds (https://github.com/Noctifer-de-Mortem/nocts_cata_mod/blob/master/nocts_cata_mod_BN/Weapons/c_ammo.json#L69) from Cataclysm++ could be a way around this. This basically makes it in the same vein as blunderbuss ammo, in how it explicitly allows multiple powder types.

One potential improvement that could be made here: adding an "overpressure" ammo effect, adding basically a chance of damaging the weapon when fired (ideally based on reliability).

Otherwise, it could simply be set so that they can use multiple ammo types, letting the player either feed them the legit version or a more ersatz option that uses more accessible materials. Blunderbusses could also potentially be converted to use either blunderbuss ammo (maybe with some rebalancing) or paper cartridges.

**Comment by RodriguesCIA** [comment]
I don't find black gunpowder difficult to craft at all. The recipe is auto learned, which takes a bit of grinding, but with crafting experience you can mostly get there in a matter of a day or two. The traditional recipe, with sulfur and saltpeter and charcoal. These materials are also abundant and relatively easy to acquire. With some luck, you get them all in a mine, having to fight off a dozen of zombie miners or so, who are comparable to just regular old plain zombies.

After all of that getting ammo you make out of paper is a sweet deal. However, I think flintlocks as the player weapon very quickly fall out, so it is a good thing that the weapons and ammo are easy to craft early on. Now, a handmade revolver though, could see some decent use through year one and perhaps even later if you are really stingy for ammo. The baseline tech level to make the ammo and gun are higher though, which, I think, would make it fair.

My point, in conclusion, is that black gunpowder is ultimately trivial to make. Anything that keeps you out of the towns is much, much safer.

**Comment by chaosvolt** [comment]
Fair point, yeah. I hadn't really given much thought to the impact of making gunpowder autolearn and hadn't really seen for myself how it affects access to it, so it's probably fine to not bother with adding overpressure loads (though that is a nifty simplification).

Other than that, the only potential complication is paper cartridges and blunderbuss rounds are autolearned recipe, but cap and ball rounds aren't.

**Comment by RodriguesCIA** [comment]
Well that is not too big of a deal in my opinion either. Even if the cartridges were auto-learned, and you managed to a full forge and tools setup without entering a town (going ham on a highway car I guess, or a Mine parking lot), you will still need something that you get in towns to make the ammo: oxidizer powder. So as you will have to get there anyway, you can raid some periphery targets that you are likely to find even in tiny towns, like grocery stores and maybe pharmacies. I am pretty sure among the magazines you can get one that teaches to make ball and cap revolver rounds.
